module/data/chat-message/actorRoll.mjs

```diff
@@ -48,17 +59,27 @@ export default class DHActorRoll extends foundry.abstract.TypeDataModel {
         return fromUuidSync(this.source.actor);
     }
 
-    get actionItem() {
+    get itemAction() {
```

Comment:
If its that you're trying to maintain convention, wouldn't this be `actionItem`? item and actor would be my preference but that seems to be the convention here.

Reply:
Changed it to actionItem for now.